requirements.txt

```diff
@@ -0,0 +1,11 @@
+openai~=0.27.8
+pandas~=2.0.3
+sqlalchemy~=2.0
+alembic~=1.11.3
+psycopg2-binary==2.9.3
+environs
+fastapi~=0.101.1
+uvicorn~=0.23.2
+pip~=21.2.4
+httpx~=0.24.1
+pydentic
```

Comment:
set a specific pydentic version. the current version is 2.x.x and Haka api server still uses 1.x.x